### DISCUSSION TITLE
Chroma Control dropping in priority within Armoury Crate when Win+P used to change screen config

### DISCUSSION THREAD

**Discussion by r4z0rw1r3** [ChromaControl/ChromaControl#313] [Ideas]
Hi,

FIrstly, brilliant software.  Just one issue I experience, I often flick dual screen using Windows Project (Win+P) between 'Extend' and 'Main screen only' on my main PC to make my slave screen available for my work tablet.  When I switch, it seems to have the effect of putting any Asus Aura related lighting back 'first in line' (as configured in Armoury Crate), effectively overidding any settings applied via Synapse sync / Chroma Control.   The same scenario occurs if I let the PC idle and the screen timeout settings kick in (as configured in Windows power options, currently set at 15 mins), no screen saver, just straight to stand-by.

If that could be fixed, I'd be a very happy bunny.

Fantastic work,

Paul.

**Comment by andrewbabbittdev** [comment]
1. Aura priority: Cant fix, go report it in the ASUS forums as I cant fix this issue.
2. Sleep issue tracked in #37

**Reply by r4z0rw1r3** [reply]
Thanks for the response.  Apologies if I posted this in the wrong place.

1.  Absolutely, will do.
2.  I'll monitor that thread.

Paul.
